tests/python_on_whales/components/test_compose.py

```diff
@@ -334,3 +339,69 @@ def test_compose_run_detach():
 
 def test_compose_version():
     assert "Docker Compose version v2" in docker.compose.version()
+
+
+def test_compose_logs_simple_use_case():
+    docker = DockerClient(
+        compose_files=[
+            PROJECT_ROOT / "tests/python_on_whales/components/compose_logs.yml"
+        ]
+    )
+    docker.compose.up(detach=True)
+    # Wait some seconds to let the container to complete the execution of ping
+    # and print the statistics
+    time.sleep(15)
+    full_output = docker.compose.logs()
+    assert "error with my_other_service" in full_output
+    assert "--- www.google.com ping statistics ---" in full_output
+    docker.compose.down(timeout=1)
+
+
+def test_compose_logs_stream():
+    docker = DockerClient(
+        compose_files=[
+            PROJECT_ROOT / "tests/python_on_whales/components/compose_logs.yml"
+        ]
+    )
+    docker.compose.up(detach=True)
+    time.sleep(15)
+    logs = docker.compose.logs(stream=True)
+    logs = list(logs)
+    any(["error with my_other_service" in log for log in logs])
+    any(["--- www.google.com ping statistics ---" in log for log in logs])

```

Comment:
```suggestion
    assert any(["error with my_other_service" in log for log in logs])
    assert any(["--- www.google.com ping statistics ---" in log for log in logs])
```
It seems it was forgotten